com.invoiceninja.app.yml

```diff
@@ -0,0 +1,35 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/flatpak/flatpak-builder/main/data/flatpak-manifest.schema.json
+
+---
+app-id: com.invoiceninja.app
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+command: invoiceninja
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --socket=pulseaudio
+  - --share=network
+  - --filesystem=xdg-pictures
+  - --filesystem=xdg-documents
+  - --filesystem=xdg-download
+modules:
+  - name: Invoice-Ninja
+    buildsystem: simple
+    only-arches:
+      - x86_64
+    build-commands:
+      - "./build-flatpak.sh"
```

Comment:
why isn't this either upstream or a few commands inline.

Reply:
I used this template, it's how they have it setup 

https://github.com/Merrit/flutter_flatpak_example

Reply:
two wrongs don't make a right.

Reply:
Does this need to be changed? I believe I saw other apps using the same approach.

Reply:
I believe I commented the same in the reviews and it got ignored.

Reply:
My preference is to leave it as is, if needs to be inlined to be accepted please let me know.